awsrte53taskgroup.go

```diff
@@ -0,0 +1,151 @@
+package ibclient
+
+type AwsRte53TaskOperations interface {
+	CreateAwsRte53TaskGroup(aws AwsRte53TaskGroup) (*AwsRte53TaskGroup, error)
+	GetAwsRte53TaskGroup(aws AwsRte53TaskGroup) (*[]AwsRte53TaskGroup, error)
+	DeleteAwsRte53TaskGroup(aws AwsRte53TaskGroup) (string, error)
+	UpdateAwsRte53TaskGroup(aws AwsRte53TaskGroup) (*AwsRte53TaskGroup, error)
+}
+
+type AwsUser struct {
+	IBBase          `json:"-"`
+	AccountID       string `json:"account_id,omitempty"`
+	AccessKeyId     string `json:"access_key_id,omitempty"`
+	LastUsed        int    `json:"last_used,omitempty"`
+	Name            string `json:"name,omitempty"`
+	NiosUserName    string `json:"nios_user_name,omitempty"`
+	SecretAccessKey string `json:"secret_access_key,omitempty"`
+	Status          string `json:"status,omitempty"`
+}
+
+// NewAwsUser creates a new AwsUser type with objectType and returnFields
+func NewAwsUser(aws AwsUser) *AwsUser {
+	res := aws
+	res.objectType = "awsuser"
+	res.returnFields = []string{"account_id", "last_used", "name", "nios_user_name", "status"}
+	return &res
+}
+
+// GetAwsUser gets the required details of the existing user
+func (objMgr *ObjectManager) GetAwsUser(aws AwsUser) (*AwsUser, error) {
+	var res []AwsUser
+	awsRte53 := NewAwsUser(aws)
+	var err error
+	err = objMgr.connector.GetObject(awsRte53, "", &res)
+	if err != nil || res == nil || len(res) == 0 {
+		return nil, err
+	}
+
+	return &res[0], err
+}
+/*type AwsRte53Task struct {
```

Comment:
Why is this commented ?

Reply:
The structure is not used. I'll remove that.